packages/jaeger-ui/src/actions/jaeger-api.js

```diff
@@ -15,11 +15,48 @@
 import { createAction } from 'redux-actions';
 import JaegerAPI from '../api/jaeger';
 
-export const fetchTrace = createAction(
-  '@JAEGER_API/FETCH_TRACE',
-  id => JaegerAPI.fetchTrace(id),
-  id => ({ id })
-);
+export function mergeAll(id) {
```

Comment:
if this is only exported for tests, please add `// exported for tests` above this line